src/vm/jitinterface.cpp

```diff
@@ -5415,6 +5409,14 @@ void CEEInfo::getCallInfo(
 
     if (directCall)
     {
+        // Direct calls to abstract methods are not allowed
+        if (pTargetMD->IsAbstract() &&
```

Comment:
This will fetch `MethodAttrs` that is not a free operation. We are fetching the MethodAttrs on all (or almost all) paths with this change. It may be a good idea to fetch it once upfront.